### DISCUSSION TITLE
Themes

### DISCUSSION THREAD

**Discussion by johnlindquist** [johnlindquist/kit#84] [Roadmap]
## Progess
* Low priority
* Targeting after 1.0

## Process
* Sometime later this year I'll see if @vojtaholik could create a set of "official" themes

The end goal would be allowing users to create/share their own themes from the App itself. Would mean exposing the styles and css build process through the app. Probably `.kenv/themes` dir with a list of .css files.

In the distant future, it would be fun to let people customize the UI positions from the app, but I haven't put any thought into it.

**Comment by zzeleznick** [comment]
This sounds great! One quick thought here is that [`ohmyzsh`](https://github.com/ohmyzsh/ohmyzsh/wiki/Customization#overriding-and-adding-themes) approach to customization could serve as some inspo. 

I'm a happy `zsh` user and use my own custom theme (forked from [gianu](https://github.com/ohmyzsh/ohmyzsh/wiki/Themes#gianu)) to display the current time and mac battery percentage in the right prompt.

**Reply by johnlindquist** [reply]
Yeah, def want people to be able to do small mixin-like tweaks rather than managing full stylesheets. Thanks for the link, great reference.
